CMakeLists.txt

```diff
@@ -25,4 +25,10 @@ add_subdirectory(icons)
 install(FILES "com.varnamproject.Fcitx5.Addon.varnamfcitx.metainfo.xml.in" 
   RENAME com.varnamproject.Fcitx5.Addon.varnamfcitx.metainfo.xml DESTINATION ${CMAKE_INSTALL_DATADIR}/metainfo)
 
-feature_summary(WHAT ALL FATAL_ON_MISSING_REQUIRED_PACKAGES)
\ No newline at end of file
+set(CMAKE_INSTALL_LIBDIR ${CMAKE_INSTALL_PREFIX}/lib)
```

Comment:
i created symlink to /usr/lib/x86_64-linux-gnu/fcitx5/libvarnamfcitx.so  because ldd does not found library in /usr/local/lib/fcitx5 as my system ldd configuration. if works without changes  this can be ignored